### PR TITLE
boards: shields: Add EVAL-ADXL372-ARDZ accelerometer shield

### PR DESCRIPTION
**Pull request by MaureenHelm**
Adds a new shield definition for the Analog Devices EVAL-ADXL372-ARDZ
accelerometer shield. This shield provides support for an ADI ADXL372
3-axis accelerometer over an Arduino SPI connector.

Signed-off-by: Maureen Helm <maureen.helm@analog.com>

Related to #78941

cc: @vladislav-pejic @dimitrije-lilic

### CODE REVIEW

**Comment by MaureenHelm** [comment]
@kartben ping

**Review by kartben — DISMISSED** [review]
sorry thought I had approved right when it came in. thanks!

**Comment by MaureenHelm** [comment]
@ttmut @ozersa can you take a look?

File: boards/adi/apard32690/apard32690_max32690_m4.dts [hunk 1]
```diff
@@ -53,6 +53,35 @@
...
+			   <6 0 &gpio2 16 0>,	/* D0 */
+			   <7 0 &gpio2 14 0>,	/* D1 */
+			   <8 0 &gpio2 15 0>,	/* D2 */
+			   <9 0 &gpio2 13 0>,	/* D3 */
+			   <10 0 &gpio0 8 0>,	/* D4 */
+			   <11 0 &gpio0 7 0>,	/* D5 */
+			   <12 0 &gpio1 25 0>,	/* D6 */
+			   <13 0 &gpio1 24 0>,	/* D7 */
+			   <14 0 &gpio1 31 0>,	/* D8 */
+			   <15 0 &gpio1 30 0>,	/* D9 */
```
Comment (ozersa) [inline boards/adi/apard32690/apard32690_max32690_m4.dts:77]: Seems P1.24 and P1.25  need to be switched. Could you double check pin mapping: https://wiki.analog.com/_media/resources/eval/user-guides/02-073637-01-c.pdf

image

**Review by kartben — APPROVED** [review]

File: boards/adi/apard32690/apard32690_max32690_m4.dts (hunk 1 above)
Comment (MaureenHelm) [reply boards/adi/apard32690/apard32690_max32690_m4.dts]: Good catch. I had rev A of the schematic and you have rev C. I reviewed all of the pins again and found a couple more changes.

**Review by ttmut — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

**Pull request merged**